tezos/context/src/timings.rs

```diff
@@ -80,32 +105,95 @@ pub fn commit(
     }
 }
 
+fn get_query_kind(name: &[u8]) -> Option<QueryKind> {
```

Comment:
@sebastiencs did this really make any improvement? I didn't consider that mapping from string to enum to be expensive

Reply:
@tizoc It makes quite a difference yes. Could we use integers (from ocaml to rust) instead of strings to determine the query name ?

Reply:
Yes, we can, or an enum. I assumed the compiler was going to be able to do something for such small strings, but I see it is not the case.

Reply:
@tizoc @sebastiencs 
I would prefer to do it like enum right now :), it is a part of ffi interface, which should be clear and exact instead of that "first byte and length" comparision like this:
```
 b'f' => {
            if name.len() == 3 {
                Some(QueryKind::Find)
            } else {
                Some(QueryKind::FindTree)
            }
        }
```